### DISCUSSION TITLE
Nvchad not working

### DISCUSSION THREAD

**Discussion by Marcodoro** [NvChad/NvChad#1940] [Q&A]
when i run this command: git clone https://github.com/NvChad/NvChad $HOME\AppData\Local\nvim --depth 1 && nvim. it says that is cloning into the repo. but the when its finished it opens nvim. but nvim is not different its still the old boring neovim nothing changed. its been hours and hours and i cant get it to work it watched the documentation and the documenatation doesnt work i tried watching youtube but it still doenst work but with my linux pc i just had to paste the git clone command and then it work.i dont really know what to do now.
![Screenshot 2023-04-18 183551](https://user-images.githubusercontent.com/118012046/232844929-b58e8e3f-21dc-4ad0-a4d8-e70bd67183be.png)

**Comment by siduck** [comment]
hi @Marcodoro , read https://github.com/NvChad/nvchad.github.io/issues/144. will fix the docs soon

**Comment by Marcodoro** [comment]
![Screenshot 2023-04-18 193943](https://user-images.githubusercontent.com/118012046/232859579-d4a3003b-f53a-45a8-a47d-2fc04c8a9ffd.png)
this is what happends when i open nvim on git bash because powershell gives me an error when i paste the git clone command
